src/main/java/com/autotune/experimentManager/utils/EMConstants.java

```diff
@@ -23,10 +23,9 @@ private EMConstants() { }
 
     public static class APIPaths {
         private APIPaths() { }
-
-        public static final String CREATE_EXPERIMENT = ServerContext.ROOT_CONTEXT + "createExperiment";
-        public static final String GET_EXPERIMENTS = ServerContext.ROOT_CONTEXT + "getExperiments";
-        public static final String GET_TRIAL_STATUS = ServerContext.ROOT_CONTEXT + "getTrialStatus";
+        public static final String CREATE_EXPERIMENT = ServerContext.ROOT_CONTEXT + "createExperimentTrial";
+        public static final String LIST_EXPERIMENTS = ServerContext.ROOT_CONTEXT + "listExperimentTrials";
```

Comment:
Is listExperimentTrials an EM API? There was a discussion on listExperiments in RM API - https://github.com/kruize/autotune/discussions/247

Reply:
As discussed removing `listExperimentTrials` as it's functionality is being implemented in `listTrialStatus`